backend/schedulers/monthly_dump_scheduler.py

```
@@ -0,0 +1,60 @@
+class MonthlyDumpScheduler:
+    """Scheduler for processing monthly dump queue jobs."""
+
+    def __init__(self) -> None:
+        self.scheduler = AsyncIOScheduler()
+        self.queue_service = MonthlyDumpQueueService()
+        self.is_running = False
+        self.interval_minutes = MONTHLY_DUMP_INTERVAL_MINUTES
+
+    def start(self) -> None:
+        if self.is_running:
+            logger.warning("Monthly dump scheduler is already running", {"job_id": "process_monthly_dump_queue"})
```

Ruff (0.15.10)

[error] Too many arguments for `logging` format string

(PLE1205)